src/status_im/config.cljs

```diff
@@ -25,6 +25,8 @@
 (goog-define ALCHEMY_OPTIMISM_GOERLI_TOKEN "")
 (goog-define ALCHEMY_OPTIMISM_SEPOLIA_TOKEN "")
 (goog-define WALLET_CONNECT_PROJECT_ID "87815d72a81d739d2a7ce15c2cfdefb3")
+(goog-define MIXPANEL_APP_ID "3350627")
+(goog-define MIXPANEL_TOKEN "5c73bda2d36a9f688a5ee45641fb6775")
```

Comment:
It's better I believe to use `goog-define` only to set up an initial value, the empty string. But the token value itself we should remove. The app ID will also be different between environments and for different devs. Does that make sense to you?

Reply:
Yes, for now I would keep it as it is, it will require infra work (maybe very little) to set up the secrets (they are not secrets btw), and it's the easiest for allowing qa to test the PR, but I will remove them eventually as I don't think we want to track metrics on dev builds, and we might want as you point out separate environments etc